testing/laszip/APKBUILD

```diff
@@ -0,0 +1,35 @@
+# Contributor: Bradley J Chambers <brad.chambers@gmail.com>
+# Maintainer: Bradley J Chambers <brad.chambers@gmail.com>
+pkgname=laszip
+pkgver=2.2.0
+_commitid=e7065cbc5bdbbe0c6e50c9d93d1cd346e9be6778
+pkgrel=0
+pkgdesc="LASzip point cloud compression library"
+url="https://github.com/laszip/laszip"
+arch="all"
+license="LGPL2+"
+depends="libgcc libstdc++ musl"
+makedepends="cmake"
+install=""
+subpackages="$pkgname-dev"
+source="$pkgname-$pkgver.tar.gz::https://github.com/LASzip/$pkgname/archive/$_commitid.tar.gz"
```

Comment:
Should be like this:
`source="$pkgname-$pkgver.tar.gz::https://github.com/LASzip/$pkgname/archive/v$pkgver.tar.gz"`